codespell_lib/data/dictionary.txt

```diff
@@ -382,6 +382,7 @@ accoriding->according
 accoring->according, occurring,
 accoringly->accordingly
 accorndingly->accordingly
+accorting->according
```

Comment:
Can you add accort as another typo too and the accordance and accorded variants.